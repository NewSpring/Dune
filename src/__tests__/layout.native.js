import { Text } from "react-native";
import renderer from "react-test-renderer";
import Layout from "../layout.native";

const testProps = [
  {
    path: "Counter",
    id: "1",
    Component: () => <Text>Hello World</Text>,
  },
  {
    path: "HelloWorld",
    id: "2",
    Component: () => <Text>Hello World Redux</Text>,
  },
];

describe("layout native", () => {
  it("works", () => {
    const tree = renderer.create(<Layout components={testProps} />);
    expect(tree).toMatchSnapshot();
  });
});
