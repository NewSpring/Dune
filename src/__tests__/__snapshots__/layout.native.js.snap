// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout native works 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello World
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello World Redux
  </Text>
</View>
`;
